File: b/Python/Disk.py
class Disk:
	# Konstruktor
	def __init__(self):
		self.__type = ""
		self.__capacity = 0
		self.__price = 0

	# Setter & Getter
	def setType(self, Type):
		self.__type = Type

	def setCapacity(self, capacity):
		self.__capacity = capacity

	def setPrice(self, price):
		self.__price = price

	def getType(self):
		return self.__type

	def getCapacity(self):
		return self.__capacity

	def getPrice(self):
		return self.__price

	# Method
	def printDisk(self):
		print("- DISK -")
		print("Type        :", self.__type)
		print("Capacity    :", self.__capacity, "GB")

File: b/Java/Disk.java
public class Disk {
	// Atribut
	private String type;
	private int capacity;
	private int price;

	// Konstruktor
	public Disk() {
		type = "";
		capacity = 0;
		price = 0;
	}

	// Setter & Getter
	public void setType(String type) {
		this.type = type;
	}
	public void setCapacity(int capacity) {
		this.capacity = capacity;
	}
	public void setPrice(int price) {
		this.price = price;
	}

	public String getType() {
		return type;
	}
	public int getCapacity() {
		return capacity;
	}
	public int getPrice() {
		return price;
	}

	// Method 
	public void printDisk() {
		System.out.println("- DISK -");
		System.out.println("Type        : " + type);
		System.out.println("Capacity    : " + capacity + " GB");
	}
}